lib/services/elasticsearch.js

```diff
@@ -678,6 +686,10 @@ class ElasticSearch extends Service {
       body: {}
     };
 
+    if (collection === HIDDEN_COLLECTION) {
+      // @todo throw `collection_reserved` error
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)